src/main/java/com/bina/varsim/fastqLiftover/readers/ARTFastqAlnReader.java

```diff
@@ -6,42 +6,60 @@
 
 import java.io.BufferedReader;
 import java.io.IOException;
+import java.util.logging.Logger;
 import java.util.regex.Pattern;
 
 public class ARTFastqAlnReader {
+    private final static Logger log = Logger.getLogger(ARTFastqAlnReader.class.getName());
     private final static Pattern nonACGTPattern = Pattern.compile("[^acgtACGT]");
     private BufferedReader fastqBr;
     private ArtAlnReader artAlnR;
     private boolean forceFiveBaseEncoding;
+    private long fqLineCount;
 
     public ARTFastqAlnReader(final BufferedReader alnR, final BufferedReader fastqBr, final boolean forceFiveBaseEncoding) throws IOException {
         this.fastqBr = fastqBr;
         this.artAlnR = new ArtAlnReader(alnR);
         this.forceFiveBaseEncoding = forceFiveBaseEncoding;
+        //assume no one has read from the above BufferedReaders
+        //this might not be true...
+        fqLineCount = 0;
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/io/LineNumberReader.html might be useful here. Would require a change of the interface here, though.